java/src/jmri/jmrix/cmri/serial/diagnostic/DiagnosticBundle.properties

```diff
@@ -0,0 +1,61 @@
+TestTypeTitle               = Test Type
+TestNodeTitle               = Test Node
+ButtonOutputTest            = Output Test
+ButtonInvert                = Invert
+ButtonWraparoundTest        = Wraparound Test
+ButtonRun                   = Run
+ButtonContinue              = Continue
+OutCardLabel                = Output Card:
+OutCardToolTip              = Enter Output Card number, numbering from 0.
+ObservationDelayLabel       = Observation Delay:
+ObservationDelayToolTip     = Enter delay (in milliseconds) between changes of output LEDs.
+InCardToolLabel             = Input Card:
+InCardToolTip               = Enter Input Card number, numbering from 0.
+InputBitsAnd                = input bits and
+OutputBits                  = output bits
+InputCard                   = input card
+OutputCard                  = output card
+InputsTitle                 = inputs
+OutputsTitle                = outputs
+CardTitle                   = Card
+BitsPerCard                 = bits per card
+FilteringDelayLabel         = Filtering Delay:
+FilteringDelayToolTip       = Enter Delay (milliseconds) if Input Card has filtering, else enter '0'.
+InvertToolTip               = Enable if data bit value should be inverted
+TestSetUpTitle              = Test Set Up
+StatusTitle                 = Status
+StatusLine1                 = Please ensure test hardware is installed.
+StatusLine2                 = Select Test Type, enter Test Set Up information, then click [{0}] below.
+ContinueTestToolTip         = Continue Current Test
+StopToolTip                 = Stop Test
+RunTestToolTip              = Run Test
+StatusRunningWraparoundTest = Running Wraparound Test
+DiagnosticError1            = Error - Bad character in Node (UA) field, please try again.
+DiagnosticError2            = Error - Node (UA) is not between 0 and 127, please try again.
+DiagnosticError3            = Error - Unknown address in Node (UA) field, please try again.
+DiagnosticError4            = Error - Bad character in Out Card field, please try again.
+DiagnosticError9            = Error - Bad character in In Card field, please try again.
+DiagnosticError5            = Error - Out Card is not between 0 and {0}, please try again.
+DiagnosticError6            = Error - Out Card is not an Output Card in your Node definition.\nPlease try again.
+DiagnosticError7            = Error - Out Card is not 0 or 1, please try again.
+DiagnosticError8            = Error - Bad character in Observation Delay field, please try again.
+DiagnosticError10           = Error - In Card is not between 0 and {0}, please try again.
+DiagnosticError11           = Error - In Card is not an Input Card in your Node definition.\nPlease try again.
+DiagnosticError12           = Error - In Card not 2 for SMINI, please try again.
+DiagnosticError13           = Error - Bad character in Filtering Delay field, please try again.
+StatusLine3                 = Port {0}, Bit {1} is ON - Compare LED's with the pattern below
```

Comment:
The Bundle classes "search up", so when a string isn't found in a particular bundle, it's then searched in the parent (in this case, cmri.serial.Bundle), etc. That lets us move common phrases up, so that they don't have to be translated multiple times.  If there are redundant ones here, it would be could to delete them so they get picked up from above.  Not urgent, but the translators will thank you.

Reply:
Thanks for the clarification on the search up.  I'll merge the added strings into the parent and remove the new bundle. This will be done in the next release.